assets/src/containers/App.js

```diff
@@ -1,32 +1,45 @@
-import React, { useState } from 'react'
+/* global myla_globals */
+
+import React from 'react'
 import { Route, withRouter } from 'react-router-dom'
-import { matchPath } from 'react-router';
+import { matchPath } from 'react-router'
 import GoogleAnalyticsTracking from '../components/GoogleAnalyticsTracking'
 import CourseList from './CourseList'
 import Course from './Course'
 
+const enrolledCourses = (myla_globals.user_courses_info.length !== 0)
+  ? JSON.parse(myla_globals.user_courses_info)
+  : ''
+
+/*
+Frozen to prevent unintentional changes to this object. This object is strictly readonly.
+myla_globals should ONLY be accessed in App.js, and nowhere else.
+*/
+const user = Object.freeze({
```

Comment:
I decided that it is preferable to access the `myla_globals` variable just from the `App.js` file, and created a frozen object that can't be modified accidentally by some child component.